sale_margin_security/models/sale_order.py

```diff
@@ -6,16 +6,20 @@
 
 class SaleOrder(models.Model):
     _name = "sale.order"
-    _inherit = ["sale.order", "product.cost.security.mixin"]
+    _inherit = ["sale.order"]
```

Comment:
issue: undo this change please. Otherwise, users with read-only permissions for cost will actually be able to write the fields.

Reply:
I'll check that

Reply:
@yajo In this case, sale order has no field that should be protected by cost group, correct? Only lines should (as it is still now)

Reply:
Ah! you're totally right.